### ISSUE TITLE
progress event not working for send file data event

### ISSUE THREAD

**Issue by jitchavan** [visionmedia/superagent#840]
i am trying to use attach event but it is not working chrome. so i am using below code for file upload which is working fine.

But it does not supporting progress event on send event for file.  

```
    const req = request.post(`https://api.cloudinary.com/v1_1/${settings.CLOUDINARY_ACCOUNT_NAME}/image/upload`);

    const data = new FormData();
    files.forEach((file) => {
      data.append('file', file);
    });
    data.append('api_key', settings.CLOUDINARY_API_KEY);
    data.append('timestamp', Date.now() / 1000);
    data.append('upload_preset', settings.CLOUDINARY_UPLOAD_PRESET);

    req.send(data);
    req.on('progress', function(e) {
                $('#'+this.filename).width(e.percent+'%');
            });
    req.end((err, res) => {
      if (err) {
        console.log('error', err);
      } else {
        console.log('success', res);
      }
    });
```

**Comment by pornel** [comment]
I think it's been fixed in 2.3.0

**Comment by halt-hammerzeit** [comment]
I confirm that this has been fixed and is working for file upload in Chrome
